src/Smarty.php

```diff
@@ -415,6 +415,13 @@ class Smarty extends \Smarty\TemplateBase {
 	 */
 	public $escape_html = false;
 
+	/**
+	 * disabled autoescape (when set to true, the next variable output is not escaped)
+	 *
+	 * @var boolean
+	 */
+	public $raw_output = false;
```

Comment:
Just one more thing: I believe we should move this to `Smarty\Compiler\Template`. Placing it in Smarty\Smarty as a public property exposes it to the users and suggests this is a setting one case use instead of a compile time flag. If you want I'll do it for you.

Reply:
I understand and I agree.
And yes, I think you will do it better than me ^^'

Reply:
Cool, did it.